scan/lib/scan/options.rb

```diff
@@ -303,7 +303,6 @@ def self.available_options
                                      optional: true,
                                      description: "The bundle identifier of the app to uninstall (only needed when enabling reinstall_app)",
                                      code_gen_sensitive: true,
-                                     default_value: ENV["SCAN_APP_IDENTIFIER"] || CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier),
```

Comment:
@revolter Did you not want to keep the `CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier)`? 🤔